src/layouts/styles.sass

```diff
@@ -0,0 +1,2 @@
+
+@import "~bulma"
```

Comment:
I avoided importing the complete module because it was overriding typography.js styles